Gopkg.toml

```diff
@@ -64,7 +64,7 @@
 [[constraint]]
   name = "github.com/prometheus/prometheus"
   #version = "2.0.0"
-  branch = "fabxc"
```

Comment:
is `queryparams` on top of `fabxc` branch? Can we here add some comment what is in this branch and what PR we are waiting on to move to master someday?

Reply:
No, fabxc stuff was merged into master already. Added comment.